core/tests/webdriverio_desktop/additionalEditorFeatures.js

```diff
@@ -371,6 +371,7 @@ describe('Full exploration editor', function() {
     await explorationEditorMainTab.setInteraction('EndExploration');
     await explorationEditorMainTab.deleteInteraction();
     await explorationEditorPage.navigateToPreviewTab();
+    await explorationEditorPage.waitForPreviewTabToLoad();
```

Comment:
This seems redundant since `navigateToPreviewTab()` already includes `waitFor.visibilityOf(previewTabLearnerViewCard, ...`. Why do we need to wait for the `previewTabLearnerViewCard` twice?